schema/request-representation.yaml

```diff
@@ -0,0 +1,58 @@
+$schema: http://json-schema.org/draft-04/schema
+title: Panda Request Representation
+description: >
+  A standard format for expressing all of the dimensions of an HTTP request in
```

Comment:
Not sure what is going on here. Does this go into the API definition? Why does it have it's own schema?